### ISSUE TITLE
Roscore does not start

### ISSUE THREAD

**Issue by RobinAmsters** [RobinAmsters/ROS_workshop#23]
When the .basrc file is configured for use with the Turtlebot (see [ROBOTIS wiki](http://emanual.robotis.com/docs/en/platform/turtlebot3/pc_setup/#network-configuration) for instructions), roscore will only start when connected to the Turtlebot (ROS is trying to start a roscore on an IP-address that does not exist any more).

**Suggestion:**
Make a script that makes switching between a networked and a local ROS configuration easier

**Comment by ratking82** [comment]
You can consider setting up `alias` and putting them in your `.bashrc`. Something like below.
```
alias set_local_master="export ROS_MASTER_URI=..... && export ROS_HOSTNAME=...."
alias set_remote_master="export ROS_MASTER_URI=..... && export ROS_HOSTNAME=...."
```
Then you can just call the alias as a cmd to set the environment variables as required.

**Comment by ratking82** [comment]
@RobinAmsters  I think this has been resolved. Could we close this issue ?

**Comment by RobinAmsters** [comment]
Yes, this was resolved with #34. Closing
